macosx/build.sh

```diff
@@ -52,16 +53,18 @@ fi
 #
 # Build archs one by one.
 #
-for arch in $ARCHS; do
+for triple in $ARCH_TRIPLES; do
+    arch=$(echo $triple | cut -d- -f1)
```

Comment:
If you want to avoid a fork here, `arch=${triple%%-*}` does the same thing.

Reply:
I personally find the bash string-substitution syntax harder to read, and the fork isn't that costly, so I prefer to leave this one as-is.